### DISCUSSION TITLE
story/plot ideas

### DISCUSSION THREAD

**Discussion by kodriq** [jeffaroundtheworld/2024gamecollab#10] [General]
ideas suggestions and discussion

**Comment by mya1919** [comment]
an escape game could be cool
with a time limit

**Comment by Jackcarlisle1** [comment]
we could lowkey have this the same typa game as Granny were you are trapped and have to get out while avoiding the enemy/monster instead of level based. Or we could do it like level or stage based with a respawn. or not quite open world but you can go wherever like BioShock.

**Reply by mya1919** [reply]
i like the avoiding an enemy thats fun

**Reply by kodriq** [reply]
yeah and there could be cool mechanics that tell you when its near

**Reply by Rhyswatsonn** [reply]
maybe a audio que like dead by daylight. I remember programming a tracking enemy is difficult but we can prolly figure it out

**Reply by ediehouston** [reply]
yeah I'm thinking we could use pathfinding since we did that last year, and I think it would be really cool if the enemy made a sound when it is near the player, I'm not sure how to do that though

**Reply by kodriq** [reply]
i feel like it making a sound wouldnt be too hard as
1. we could make like a "circle" around it and when the player enters that it makes a sound
2. its always making a sound but it obv gets louder when ur closer or smth

**Reply by ediehouston** [reply]
Yeah I'm going to do some research on how we can execute a sound when the enemy has entered the "circle" of the player
